### ISSUE TITLE
Export Training Data from Firebase to Cloud Data Storage

### ISSUE THREAD

**Issue by rosst** [Welthungerhilfe/cgm-ml#16]
**Business Requirement:**

As a data scientist working on the Child Growth monitor I want an ETL job that extracts all the data I need to run the model from Firebase and stores it in a cloud storage bucket so that I can immediately start working on the model and don't have to store the full data set.

**Tasks**

Whole process runs in a container on the existing cloud infrastructure. 

1. A reader process pulls the personal, measurement and PCD data from Firebase
2. The ETL process extracts QrCode and measurement data.
3. A writer process writes the QrCode, measurement and point cloud data to cloud storage bucket.
    
**Acceptance Criteria**

1. Data required to train the model is extracted automatically from Firebase and saved to cloud storage.
2. Data is versioned
3. ETL job is run daily.
4. Data format is as per existing data generator: <QrCode, PCD, measurement (height, weight)>

**Comment by ashish-bagri** [comment]
@mmatiaschek @rosst Should the splitting of the data into training and test set be part of the ETL (as is currently in the data generator) ? IMHO this should be left to the data scientist and ETL should simply create the entire dataset
